pkg/prometheus/promcfg.go

```diff
@@ -1617,7 +1616,7 @@ func getLimit(user *uint64, enforced *uint64) *uint64 {
 	}
 
 	if user == nil {
-		return enforced
+		return user
```

Comment:
I'm confused by this change.

Reply:
I just pushed another version, can you take a look?